### PR TITLE
feat: Pagopa selfcare

### PR DESCRIPTION
**Pull request by andrea-putzu**
### List of changes

added new backend to deploy in k8s

### Motivation and context

adding pagopa backoffice for selfcare

### Type of changes

- [X] Add new pipeline
- [ ] Update pipeline configuration
- [ ] Remove pipeline

### :warning: If it's new pipeline with code review have you added pagopa-github-bot -> Role: admin?

- [X] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: azure-devops/shared/06_pagopa-selc-backoffice-backend.tf [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+module "pagopa-selc-backoffice-backend_code_review" {
+  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_code_review?ref=v2.0.4"
+  count  = var.pagopa-selc-backoffice-backend.pipeline.enable_code_review == true ? 1 : 0
+
+  project_id                   = data.azuredevops_project.project.id
+  repository                   = var.pagopa-selc-backoffice-backend.repository
+  github_service_connection_id = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_github_pr_id
+  # path                         = "${local.domain}\\pagopa-selc-backoffice-backend"
```
Comment (pasqualespica) [inline azure-devops/shared/06_pagopa-selc-backoffice-backend.tf:85]: it'd be good to segregate pipelines in a dedicate project folder.
why was it commented?

File: azure-devops/shared/06_pagopa-selc-backoffice-backend.tf [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+module "pagopa-selc-backoffice-backend_deploy" {
+  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_deploy?ref=v2.0.4"
+  count  = var.pagopa-selc-backoffice-backend.pipeline.enable_deploy == true ? 1 : 0
+  
+  project_id                   = azuredevops_project.project.id
+  repository                   = var.pagopa-selc-backoffice-backend.repository
+  github_service_connection_id = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_github_rw_id
+  #path                         = "${local.domain}\\pagopa-selc-backoffice-backend"
```
Comment (pasqualespica) [inline azure-devops/shared/06_pagopa-selc-backoffice-backend.tf:112]: ditto

File: azure-devops/shared/06_pagopa-selc-backoffice-backend.tf [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    # prod_container_namespace = "pagopapcommonacr.azurecr.io"
+  }
+  
+  # deploy secrets
+  pagopa-selc-backoffice-backend-variables_secret_deploy = {
+
+  }
+}
+
+module "pagopa-selc-backoffice-backend_code_review" {
```
Comment (pasqualespica) [inline azure-devops/shared/06_pagopa-selc-backoffice-backend.tf:78]: Selfare do you need CI pipeline too ?

**Review by pasqualespica — CHANGES_REQUESTED** [review]

File: azure-devops/shared/06_pagopa-selc-backoffice-backend.tf (hunk 3 above)
Comment (andrea-putzu) [reply azure-devops/shared/06_pagopa-selc-backoffice-backend.tf:78]: selfcare need both code review and deploy

File: azure-devops/shared/06_pagopa-selc-backoffice-backend.tf (hunk 1 above)
Comment (andrea-putzu) [reply azure-devops/shared/06_pagopa-selc-backoffice-backend.tf:85]: do you prefer to move this file into pagopa-azure-devops/azure-devops/selfcare?

**Review by pasqualespica — DISMISSED** [review]
LGTM 👍 i suggest to add path directory project too.

**Review by pasqualespica — APPROVED** [review]
LGTM 💯

**Pull request merged**